.github/version.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+import argparse
+from subprocess import check_output
```

Comment:
don't use "from" imports, import the module itself and refer to its contents, i.e. `subprocess.check_output`